kubernetes/services/concent-api-worker.yml.j2

```diff
@@ -19,11 +19,27 @@ spec:
             - containerPort: 80
           resources:
             requests:
-              memory: "{{ resource_limits.worker_apps.concent_api_worker.memory.requests }}"
-              cpu:    "{{ resource_limits.worker_apps.concent_api_worker.cpu.requests }}"
+              memory: "{{
+                        ((((concent_versions[concent_version].resource_limits | default({})).worker_apps
+                        | default({})).concent_api_worker | default({})).memory | default({})).requests
+                        | default('{{ resource_limits.worker_apps.concent_api_worker.memory.requests }}', true)
```

Comment:
I think that even getting rid of defaults would be better than this. I.e. (if the idea with a filter does not work out), just define these values separately for every cluster in its config instead of inheriting them from `var.yml`.